libs/cache/CMakeLists.txt

```diff
@@ -5,8 +5,6 @@
 
 cmake_minimum_required(VERSION 3.3.2 FATAL_ERROR)
 
-project(HPX.cache CXX)
-
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
 include(HPX_AddDefinitions)
```

Comment:
One last nitpick: this is not needed anymore. Instead we should add a `include(HPX_Message)` now. After that this PR is good to go, I think. Thanks a lot!

Reply:
Thank you !